src/content/post/drush-make-gitignore.md

```diff
@@ -0,0 +1,52 @@
++++
+date        = "2016-04-08T01:00:00-04:00"
```

Comment:
In order to achieve the folllowing path scheme: year/month/day/post-name the post should be placed within the directory tree, that should be created if not existent.

I don:t know if the slug can be enough.

Can you check in ?